### PR TITLE
[7.2.x] JBPM-5531: Problems after deleting a form that is used as a subform in other form.

### PR DESCRIPTION
**Pull request by pefernan**
This changes are introduced by: kiegroup/kie-wb-common#1009

@manstis @jsoltes  could you review?

This PR is part of an ensemble, please merge with:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/490
https://github.com/kiegroup/kie-wb-common/pull/1009
https://github.com/kiegroup/drools-wb/pull/561
https://github.com/kiegroup/optaplanner-wb/pull/192
https://github.com/kiegroup/kie-wb-distributions/pull/563
https://github.com/kiegroup/jbpm-wb/pull/817
https://github.com/kiegroup/jbpm-form-modeler/pull/128

### CODE REVIEW

File: kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml [hunk 1]
```diff
@@ -72,6 +72,7 @@
   <inherits name="org.kie.workbench.common.services.KieWorkbenchCommonServicesAPI"/>
   <inherits name='org.kie.workbench.common.services.datamodel.KieWorkbenchCommonDataModelAPI'/>
   <inherits name='org.kie.workbench.common.services.refactoring.KieWorkbenchCommonRefactoringAPI'/>
```
Comment (manstis) [inline kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml]: You remove this in ```KIEWebapp.gwt.xml``` want to do the same here (I assume the "client" one inherits the "api" one)

**Comment by pefernan** [comment]
@manstis better now?

**Comment by manstis** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Comment by pefernan** [comment]
Jenkins please execute full downstream build.

**Comment by pefernan** [comment]
jenkins please execute full downstream build

**Comment by pefernan** [comment]
jenkins please execute full downstream build

**Comment by pefernan** [comment]
@psiroky I don't understand why is it failing... it seems that when building the jbpm-form-modeler it tries to build the master instead of the JBPM-5531-remove-forms branch (which is a 7.2.x)

**Comment by pefernan** [comment]
jenkins please execute full downstream build

**Comment by pefernan** [comment]
jenkins please execute full downstream build

**Comment by pefernan** [comment]
jenkins please execute full downstream build

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by pefernan** [comment]
@manstis wow three greens after 10 days! I see that there's a new 7.3 branch... Should I create the PR for 7.3?

**Comment by manstis** [comment]
@pefernan Yay! (Aren't you on PTO?) Apparently you can re-target existing PRs for a new branch (IDK how). All these 7.2.x PRs are pretty much dead (as 7.2.0.Final is almost released.. but I can still merge - even if it just gives us a warm feeling of completion)

**Comment by pefernan** [comment]
@manstis yes I'm on PTO but I think I need to do some work... That would be like taking PTO from my family.

**Pull request merged**